examples/cms-buttercms/.env.sample

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_BUTTER_CMS_API_KEY=your_auth_token
```

Comment:
I think we should keep this as .env.local.example - any reason it changed?

Reply:
reverted back to .env.local.example